e2e-tests/gef/cypress/integration/unissued-facilities-change-all-MIN.spec.js

```diff
@@ -67,6 +67,7 @@ context('Unissued Facilities MIN - change all to issued from unissued table', ()
       applicationPreview.unissuedFacilitiesReviewLink().contains('View unissued facilities');
       applicationPreview.submitButtonPostApproval().should('not.exist');
       applicationPreview.mainHeading().contains(CONSTANTS.DEAL_SUBMISSION_TYPE.MIN);
+      applicationPreview.automaticCoverSummaryList().contains('No - submit as a manual inclusion application');
```

Comment:
Should we update both e2e tests to maker sure that eligibility questions and answers are displayed as well? Checking for "is EC list rendered" is probably enough rather than checking list items

Reply:
done